src/NuGet.Clients/VsConsole/Console/OutputConsole/Guids.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGetConsole
+{
+    internal static class GuidList
```

Comment:
There's another one `VsConsole/Console/Guids.cs`. Take what's really needed and copy there.